alpha-build-machinery/make/examples/multiple-binaries/Makefile.test

```diff
@@ -1,8 +1,21 @@
+SHELL :=/bin/bash -euo pipefail
+
 all:
 	$(MAKE) -C . build
 	[[ -f ./openshift ]]
 	[[ -f ./oc ]]
 
+	# test version is set correctly when linking
+	#   majorFromGit, minorFromGit are deprecated upstream and set to empty value
+	#   we avoid comparing time to avoid flakes
+	diff <( ./oc | sed '$$d' ) <( \
+		echo ""; \
+		echo ""; \
+		echo $$( git rev-parse --short "HEAD^{commit}" 2>/dev/null ); \
```

Comment:
why echo and not directly the git command?

Reply:
Same everywhere else.

Reply:
say `rev-parse` returns `4d798104` then you'd get `/bin/bash: 4d798104: command not found`

Reply:
no. Just remove ech and the `$$( ... )`

Reply:
```
diff <( ./oc | sed '$$d' ) <( \
		echo ""; \
		echo ""; \
		git rev-parse --short "HEAD^{commit}" 2>/dev/null; \
...
)
```

Reply:
sorry, I forgot to remove the subshell when trying your suggestion